gitlab.go

```diff
@@ -52,6 +52,22 @@ const (
 	OwnerPermission      AccessLevel = 50
 )
 
+// NotificationLevel represents a notification level within Gitlab.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/...?
+type NotificationLevel int
+
+// List of available notification levels
+//
+// GitLab API docs: http://doc.gitlab.com/ce/...?
+const (
+	DisabledNotifications NotificationLevel = 0
+	ParticipatingNotifications NotificationLevel = 1
+	WatchNotifications NotificationLevel = 2
+	GlobalNotifications NotificationLevel = 3
+	MentionNotifications NotificationLevel = 4
+)
```

Comment:
Yes, it's cleaner, but you don't know quickly say which value belongs to which notification level. You have to always count lines to get value. :) And maybe in the future Gitlab developers will add new notification level and it's value for some reason will not be 5 but maybe 10.

But anyway if you wish I can replace it. Let me know.